src/modules/pasteplain/PastePlain/Helpers/OSInterop.cs

```diff
@@ -0,0 +1,1211 @@
+﻿// Copyright (c) Microsoft Corporation
```

Comment:
This is a big file. Had to introduce a lot of stuff to get `SendInput()` to work (particularly to define what an `INPUT` is).